linter/mount.go

```diff
@@ -0,0 +1,24 @@
+package linter
+
+import (
+	"fmt"
+	tooling "github.com/JetBrains/qodana-cli/v2024/linter/tooling"
+	"github.com/JetBrains/qodana-cli/v2024/platform"
+	"os"
+	"path/filepath"
+)
+
+func (o *CltOptions) MountTools(_ string, mountPath string, _ *platform.QodanaOptions) (map[string]string, error) {
+	val := make(map[string]string)
+	val["clt"] = filepath.Join(mountPath, "tools", "netcoreapp3.1", "any", "JetBrains.CommandLine.Products.dll")
+	archive := "clt.zip"
+	if _, err := os.Stat(val["clt"]); err != nil {
+		if os.IsNotExist(err) {
```

Comment:
If stat fails with an error that is not `isNotExist` we just swallow it and continue. Looking at the docs, it might also be a permission error for example - probably better to return the error in such case